cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -34,7 +37,33 @@
             'link_url': '#'
         }) }}
     </div>
-    <div class="block">
+    <div class="o-full-width-text">
+        {{ full_width_text.render( 'sublanding-page-1/_template-prototype-full-width-content.html' ) }}
+        <div class="m-inset">
+            {{ related_links.render( [
+                ['#', 'Related Link 1'],
```

Comment:
Is that necessary for two items if this is only going to be generated by the backend?